posts/2024-11-11-jupyter-book-org.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Jupyter Book Becomes a Jupyter Subproject
+subtitle: Jupyter Book joins the ranks of other Jupyter Subprojects such as JupyterLab and JupyterHub.
+date: 2024-11-11
+authors:
```

Comment:
Can't we define authors in the project YML rather than in the post? We should add an issue to add the orcid info etc of all of the JB team members

Reply:
> A new on style: I tried to remove passive voice as much as possible. @agoose77 I recommend you do so in future iterations. Academia really drills the passive voice into us, but it's not a great way of communicating in most contexts

Ouch :face_holding_back_tears:

Reply:
>  I think we should prioritize having more, shorter blog posts over fewer, longer/over-complete ones. So I'm +1 on just rolling with this one.

Hard agree -- I've been seeing the benefits of Jenny's high-throughput blog posts on 2i2c's blog, and although they're not necessarily short, I see the value.

Reply:
> Ouch 🥹

This is a judgment-free zone :-) I'm just trying to give clear feedback!

Reply:
Oh, haha, no I mean 'ouch - I've been seen'!